Fanfiction/dist/metadata.json

```diff
@@ -0,0 +1,22 @@
+{
+    "author": {
+        "name": "Nekolen",
+        "id": "270125856679002113"
+    },
+    "service": "Fanfiction",
+    "description": {
+        "en": "FanFiction | unleash your imagination"
+    },
+    "url": "www.fanfiction.net",
+    "version": "1.0",
+    "logo": "https://i1.wp.com/geezerwench.com/wp-content/uploads/2017/10/Fanfiction-net-logo-400x400.png?fit=400%2C400",
```

Comment:
low quality logo

Reply:
i had to magnify the logo because it wasnt fitting in 512*512

Reply:
still, it has a low resolution

and you need to change the version number on dueling nexus

Reply:
i changed the version number and placed logo and thumbnail properly